rbi/bible_bot.rbi

```diff
@@ -0,0 +1,213 @@
+# typed: strict
+
+module BibleBot
+  class << self
+
+    sig { returns(include: T::Boolean) }
+    def include_apocryphal_content?; end
+
+    sig { params(include: T::Boolean).void }
+    def include_apocryphal_content=(include); end
+  end
+end
+
+class BibleBot::Bible
+  class << self
+    sig { returns(T::Array[BibleBot::Book]) }
+    def books; end
+  end
+end
+
+class BibleBot::Book
+  class << self
+    sig { params(name: T.nilable(String)).returns(T.nilable(BibleBot::Book)) }
+    def find_by_name(name); end
+
+    sig { params(code: T.nilable(String)).returns(T.nilable(BibleBot::Book)) }
+    def find_by_code(code); end
+
+    sig { params(id: T.nilable(Integer)).returns(T.nilable(BibleBot::Book)) }
+    def find_by_id(id); end
+  end
+
+  sig { returns(Integer) }
+  def id; end
+
+  sig { returns(String) }
+  def name; end
+
+  sig { returns(String) }
+  def abbreviation; end
+
+  sig { returns(String) }
+  def dbl_code; end
+
+  sig { returns(T::Array[Integer]) }
+  def chapters; end
+
+  sig { returns(Symbol) }
+  def testament; end
+
+  sig { returns(String) }
+  def testament_name; end
+
+  sig { returns(T::Boolean) }
+  def apocryphal?; end
+
+  sig { returns(String) }
+  def formatted_name; end
+
+  sig { returns(String) }
+  def string_id; end
+
+  sig { returns(BibleBot::Reference) }
+  def reference; end
+
+  sig { returns(T::Array[String]) }
+  def chapter_string_ids; end
+
+  sig { returns(T::Array[Integer]) }
+  def verse_ids; end
+
+  sig { returns(BibleBot::Verse) }
+  def start_verse; end
+
+  sig { returns(BibleBot::Verse) }
+  def end_verse; end
+
+  sig { returns(T.nilable(BibleBot::Book)) }
+  def next_book; end
+end
+
+class BibleBot::Verse
+  include Comparable
+
+  class << self
+    sig { params(id: T.nilable(Integer)).returns(T.nilable(BibleBot::Verse)) }
+    def from_id(id); end
```

Comment:
Technically, can't this method accept a String as well?

Reply:
Looks like yes. I'll update this, as well as the YARD for that method.

Reply:
https://github.com/LittleLea/bible_bot/pull/19/commits/09d03c719d528077bdac5d248e3ac1ad7e0ac50a